src/idom/utils.py

```diff
@@ -289,6 +272,20 @@ def _hypen_to_camel_case(string: str) -> str:
     return first.lower() + remainder.title().replace("-", "")
 
 
+def del_html_head_body_transform(vdom: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Annotating as `VdomDict` should work now.

```suggestion
def del_html_head_body_transform(vdom: VdomDict) -> VdomDict:
```